pkg/ocm/helpers.go

```diff
@@ -273,3 +273,46 @@ func RandomLabel(size int) string {
 	}
 	return string(chars)
 }
+
+func (c *Client) LinkAccountRole(accountID string, roleARN string) error {
+	labels, err := c.ocm.AccountsMgmt().V1().Accounts().Account(accountID).Labels().List().Send()
+	if err != nil {
+		return err
+	}
+	existingARN := ""
+	if labels.Items().Len() > 0 {
+		labels.Items().Range(func(index int, item *amsv1.Label) bool {
+			if item.Key() == "sts_user_role" {
+				existingARN = item.Value()
+				return true
+			}
+			return false
+		})
+	}
+	exists := false
+	if existingARN != "" {
+		existingARNArr := strings.Split(existingARN, ",")
+		if len(existingARNArr) > 0 {
+			for _, value := range existingARNArr {
+				if value == roleARN {
+					exists = true
+					break
+				}
+			}
+		}
+	}
+	if exists {
+		return nil
+	}
+	roleARN = existingARN + "," + roleARN
```

Comment:
If there's nothing inside `existingARN` this creates a leading `,`

Reply:
Maybe something like this would be sufficient:
```
if existingARN != "" {
  roleARN = existingARN + "," + roleARN
}
```

Reply:
done.

{
  "kind": "LabelList",
  "page": 1,
  "size": 1,
  "total": 1,
  "items": [
    {
      "id": "216k1imPzssQocCxTBV5rM8V4Qo",
      "kind": "Label",
      "href": "/api/accounts_mgmt/v1/accounts/1WDBU5E101KnGowHfbeANgKzwA6/labels/sts_user_role",
      "key": "sts_user_role",
      "value": "arn:aws:iam::765374464689:role/ft3-OCM-User-pvasanth_openshift-Role",
      "internal": false,
      "type": "Account",
      "created_at": "2021-11-18T20:51:50.917434Z",
      "updated_at": "2021-11-18T20:51:50.917434Z",
      "account_id": "1WDBU5E101KnGowHfbeANgKzwA6"
    }
  ]
}